utils/validate_utils.py

```diff
@@ -0,0 +1,84 @@
+"""
+utilities for validating logistic regression models on MitoCheck single-cell dataset
+"""
+
+import pandas as pd
+import pathlib
+
+
+def create_classification_profiles(
+    plate_classifications_dir_link: str, cell_line_plates: dict
+) -> pd.DataFrame:
+    """
+    create classsification profiles for correlation to cell health label profiles
+    a classification profile consists of classification probabilities averaged across perturbation and cell line
+
+    Parameters
+    ----------
+    plate_classifications_dir_link : str
+        link to plate classifications hosted on GitHub
+    cell_line_plates : dict
+        cell line names, each with a list of plate names that are correlated to their respective cell line
+
+    Returns
+    -------
+    pd.DataFrame
+        classification profiles dataframe
+    """
+
+    cell_line_classification_profiles = []
+
+    for cell_line in cell_line_plates:
+        # create one large dataframe for all 3 plates in the particular cell line
+        cell_line_plate_names = cell_line_plates[cell_line]
+        cell_line_plate_classifications = []
+        for cell_line_plate_name in cell_line_plate_names:
+            plate_classifications_link = f"{plate_classifications_dir_link}/{cell_line_plate_name}_cell_classifications.csv.gz"
+            plate_classifications = pd.read_csv(
+                plate_classifications_link, compression="gzip", index_col=0
+            )
+            cell_line_plate_classifications.append(plate_classifications)
+        cell_line_plate_classifications = pd.concat(
+            cell_line_plate_classifications, axis=0
+        ).reset_index(drop=True)
+
+        # create dataframe with cell classifications averaged across pertubation, include cell line metadata
+        # add cell line metadata, rename pertubation column
+        cell_line_plate_classifications["Metadata_cell_line"] = cell_line
+        cell_line_plate_classifications = cell_line_plate_classifications.rename(
+            columns={"Metadata_Reagent": "Metadata_pert_name"}
```

Comment:
Would the word changes for `reagent` reduce understandability for anyone reading through the output of this project? If so, I'd suggest adding a comment or two here and possibly information within the readme(s).

Reply:
I think it could reduce readability, I added a comment to explain this change in https://github.com/WayScience/phenotypic_profiling_model/pull/7/commits/3d3a30112e529b4df3ac7857a2102a6f678a974a